app/services/settings.js

```diff
@@ -37,7 +37,7 @@ export default Ember.Service.extend({
     "stock.only_publish_singletons": true,
     "stock.only_designate_singletons": true,
     "stock.enable_box_pallet_creation": false,
-    "stock.allow_item_actions": false,
+    "stock.allow_item_actions": true,
```

Comment:
Should we maybe keep the default to `false` ? at least until StockIt still exists 🤔